admin_manual/maintenance/migrating.rst

```diff
@@ -62,3 +62,141 @@ When a user tries a URL that is not whitelisted the following error appears:
    :scale: 75%
    :alt: Error message when URL is not whitelisted
 
+=======
+Example
```

Comment:
@voroyam @settermjd Just a side-note to this (and the other Migrate) headline. These looks quite strange in the ToC:

https://doc.owncloud.org/server/10.0/admin_manual/maintenance/migrating.html

Both should be one level down or am i wrong? Additionally the "Example" headline looks incomplete